editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs

```diff
@@ -1139,6 +1183,58 @@ pub fn modulo_properties(document_node: &DocumentNode, node_id: NodeId, _context
 	vec![operand("Modulo", 1)]
 }
 
+pub fn circle_properties(document_node: &DocumentNode, node_id: NodeId, _context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
+	let operand = |name: &str, index| {
+		let widgets = number_widget(document_node, node_id, index, name, NumberInput::default(), true);
+
+		LayoutGroup::Row { widgets }
+	};
+	vec![operand("Radius", 1)]
```

Comment:
There is no real need to have an operand closure only to call it once. Could be shortened to:
```rs
let radius = number_widget(document_node, node_id, 1, "radius", NumberInput::default(), true);
vec![LayoutGroup::Row { widgets: radius }]
```